android/gradletealeaf.xml

```diff
@@ -1,6 +1,5 @@
 //<!--START_PLUGINS_DEPENDENCIES-->
 	//<!--Firebase Dependencies-->
-     implementation 'com.google.firebase:firebase-perf:16.0.0'
      implementation 'com.google.firebase:firebase-config:16.0.0'
      implementation 'com.google.firebase:firebase-invites:16.0.0'
```

Comment:
invites and config are also not used. Can be removed Anil

Reply:
@rampr Ok